frontend/src/Data/Form.ts

```diff
@@ -72,13 +73,13 @@ export const xToYFormInit: () => Map<string, FormItem> = () =>
       ["deadline",
         {
           order: 3,
-          label: "Deadline: (example: 2022-01-01T00:01:40Z)",
+          label: "Deadline:",
```

Comment:
The date picker is nice, but I think it would still be nice to give a suggestion, maybe a few minutes in the future from "now" at any given moment?